### ISSUE TITLE
BOSS error, disabled upon restart.

### ISSUE THREAD

**Issue by ChonkWars** [kangarko/Boss#446]
Hi there!

First of all I'd like to say I absolutely LOVE this plugin and I have no regret of my purchase whatsoever.
Very nice and clear plugin, no hassle installing or configurating it and easy to adopt into my server.

There's only one problem!
After putting down a spawner and configuring the mob, we wanted to config the spawn cage and after right clicking it the message; "Oops! There was a problem with this menu, please contact the administrator to review the console for details." popped up.

The plugin continued to work, even spawning the bosses as we had configured them, but after a server restart it seemed the plugin got disabled completely and all /boss commands stopped working entirely. 

We are running a slightly modded server, 1.14.4 Paper Spigot with Multicraft.

Down below is the BOSS error log.

If needed I can provide a link to our full plugin folder, since it's too big to upload here. (124 MB when zipped)

Hopefully we'll be able to resolve this problem!

Thanks in advance.

Henk, 

ChonkWars.

a

BOSS error log displays this message;
------------------------------------[ 11.09.2019 04:11:36 ]-----------------------------------
Boss 3.4.17 encountered a LPt9
Running Paper 1.14.4-R0.1-SNAPSHOT and Java 1.8.0_211
Plugins: CoreProtect v2.17.5, LuckPerms v4.4.1, AcceptRules v2.1, WorldEdit v7.0.1;61bc012, PlaceholderAPI v2.10.3, ClearLag v3.0.6, Boss v3.4.17, AdvancedBan v2.1.5-RELEASE, Votifier v1.9, ConsoleSpamFix v1.7.3, ProtocolLib v4.4.0-SNAPSHOT-b390, WorldGuard v7.0.0;02b731f, Multiverse-Core v4.1.1-SNAPSHOT-b777, VotingPlugin v5.19.4, DoOdy, DeluxeMenus v1.12.0, RandomTeleport v2.5, Multiverse-Portals v4.1.1-SNAPSHOT-b808, NoPhantoms v1.0, Vault v1.7.2-b107, PrivateMessager v1.0, MassiveCore v3.2.3, ShopGUIPlus v1.20.4, Coins v1.7.3, Essentials v2.17.1.0, Holograms v2.9.1-SNAPSHOT, Citizens v2.0.25-SNAPSHOT (build 1732), AuctionHouse v1.8.10, EssentialsChat vTeamCity, CommandNPC v1.8.5, AntiSwear v5.2, EssentialsExtras v2.15.0.1, EssentialsSpawn vTeamCity, SlotMachine v6.0.0, Factions v3.2.3
----------------------------------------------------------------------------------------------

More Information: 
Expected location (String) but got String: The World 1806 13 2353
LPt9 Report / Expected location (String) but got String: The World 1806 13 2353
	 at org.mineacademy.boss.double. .LPT2.const(SourceFile:78)
	 at org.mineacademy.boss.double. .cOM5.native(SourceFile:350)
	 at org.mineacademy.boss.double. .ay.const(SourceFile:50)
	 at org.mineacademy.boss.double. .ay.transient(SourceFile:105)
	 at org.mineacademy.boss.double. .ay.const(SourceFile:92)
	 at org.mineacademy.boss.double. .L$protected.const(SourceFile:181)
	 at org.mineacademy.boss.double. .L$protected.const(SourceFile:150)
	 at org.mineacademy.boss.double. .nUL.const(SourceFile:247)
	 at org.mineacademy.boss.double. .Com5.onMenuClick(SourceFile:76)
	 at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor136.execute(Unknown Source)
	 at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69)
	 at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80)
	 at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
	 at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:545)
----------------------------------------------------------------------------------------------

**Comment by ChonkWars** [comment]
Also please note that this only happens when we are using the spawn cages.

**Comment by ChonkWars** [comment]
Update:


Spawning works in biomes, not in WorldGuard regions and selecting regions with the tool and then executing /boss region add name does work, but won't save the regions made in the plugin.

**Comment by kangarko** [comment]
Your world contains a space in it "The World". This will cause issues with a lot of plugins, you can use your multiworld plugin to rename it so it does not contain a space.

For spawning, make sure you selected at least 1 block in the ground. If it won't save the region are you getting an error message?

**Comment by ChonkWars** [comment]
Hi!

I'll be unable to do further testing until I'm home again later today. 

I found out our world contains a space and renamed it back to just "world" in the server config file, would that be enough? 

The multiverse plugins are not being used yet, as we are still in the proces of implementing it into the server. 
For now, we only have world, the nether and the end.

It does not give an error upon saving, everything seems as normal until I execute the command /boss region list and it shows 0 regions loaded. 
When I try to see if maybe it has saved but just isnt enabled yet, so I try to add a spawn region to the mob, it also doesn't show up any region for it to spawn in other than the worldguard regions.

**Comment by ChonkWars** [comment]
Also please forgive me for any noobyness!

This is our first Minecraft server and its about one and a half week old now, we're still getting settled when it comes to the server side things.
I feel kind of stupid for renaming my world now, hehe.

**Comment by ChonkWars** [comment]
Update:


I renamed the world to "TheWorld" instead of "The World" and it fixed all issues.

Thank you.

As I said before, awesome plugin!

**Comment by ChonkWars** [comment]
Only one more question from my side really, and maybe the awnser to this question is one I should look for elsewhere, but I thought I would ask it anyway.

We have made an NPC allowing a boss to be spawned, this requires the player to have acces to the command boss.command.spawn, is there any way around this?
So that they can still use the NPC to spawn the boss we made but without them actually being able to run the command from their chat? 

Sincerely

Henk
